### ISSUE TITLE
Can`t run on your PC error (virtualbox.exe)

### ISSUE THREAD

**Issue by necros2k7** [upx/upx#411]
## What's the problem (or question)?
Compressed exe gives error

## What should have happened?
Run

## Do you have an idea for a solution?
-

## How can we reproduce the issue?
1. Install https://download.virtualbox.org/virtualbox/6.1.16/VirtualBox-6.1.16-140961-Win.exe
2. pack virtualbox.exe
3.
4.

## Please tell us details about your environment.

* UPX version used (`upx --version`): latest autobuild
* Host Operating System and version: win10 2004
* Host CPU architecture: x64
* Target Operating System and version: win10 2004
* Target CPU architecture: x64

**Comment by jreiser** [comment]
Confirming:  Using a recent tip of `devel` branch UPX git-149922+ to compress VirtualBox.exe after  installing VirtualBox-6.1.16-140961-Win.exe:
```
$ ls -l VirtualBox.exe
-rwxr-xr-x. 1 user group   2400080 Oct 16 17:34 VirtualBox.exe
$ sha256sum VirtualBox.exe
2ca3f80fe4bd0fd7035e77cf6ea92dce4aec4e53f5571d50a8bec45bc94d9e04  VirtualBox.exe
$ ../upx.out -f -o VB-upx.exe VirtualBox.exe
                       Ultimate Packer for eXecutables
                          Copyright (C) 1996 - 2020
UPX git-149922+ Markus Oberhumer, Laszlo Molnar & John Reiser   Jan 24th 2020

        File size         Ratio      Format      Name
   --------------------   ------   -----------   -----------
   2400080 ->    989008   41.21%    win64/pe     VB-upx.exe                           
```
Then running VB-upx.exe under CMD shell on 64-bit Win10 Pro 2004 (build 19041.572) produces a dialog box "This app can't run on your PC" after a few seconds which could be the de-compression by the upx stub.
I don't know what this means.

**Comment by bitraid** [comment]
@jreiser That's because the executable has set the `INTEGRITYCHECK` bit (0x80) of the `DLL Characteristics` optional header flag, which *specifies that the digital signature of the binary image must be checked at load time* (which of course is no longer valid). Clearing the flag will fix the problem, although i think it's better for upx to prevent the compression of such executables. I can open a PR if you agree.

**Comment by jreiser** [comment]
@bitraid  Yes, please open a PullRequest.  It will document and explain.  Perhaps a future command-line option might offer to override the decision not to compress.

**Comment by jreiser** [comment]
Fixed by PullRequest #413 noted above.
